.gitignore

```diff
@@ -96,3 +96,11 @@ node/generator/specifications
 node/PATH_TO_DAWN
 node/third_party/webnn-polyfill
 node/node_modules
+node/third_party
+node/package-lock.json
```

Comment:
`package-lock.json` is intended to be checked into source control for good reason, see https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5, perhaps we could do that too.